libs/icons/package.json

```diff
@@ -75,6 +75,7 @@
     "npm-run-all": "^4.1.5",
     "prettier": "^2.7.1",
     "puppeteer": "^10.0.0",
+    "run-func": "^3.0.0",
```

Comment:
This is used to run a function in a file while using node. You can read more [here](https://github.com/DVLP/run-func)